packages/node/src/integrations/anr/index.ts

```diff
@@ -153,6 +161,12 @@ async function _startWorker(
     }
   }
 
+  let debugImages: Record<string, string> = getFilenameToDebugIdMap(initOptions.stackParser);
+
+  onModuleLoad(() => {
+    debugImages = getFilenameToDebugIdMap(initOptions.stackParser);
```

Comment:
Isn't this potentially called every `require`/`import`? That means we re-calculate this every time. That feels like a lot of work.

Reply:
We could debounce the `require`/`import` but any delay sending the debug ids to the worker could result in missing debug ids. 

If `global._sentryDebugIds` is undefined, `getFilenameToDebugIdMap` returns immediately so this only impacts those with injected debug id polyfills. `getFilenameToDebugIdMap` already caches a lot. If there have been no additions to `global._sentryDebugIds`, no stack parsing occurs. There is still scope for it to cache more though and simplify the work done on each call. That would probably be a good start 🤔

Reply:
Alright let's proceed for now, we should re-visit this though.